bindings/go/ctf/ctf.go

```diff
@@ -110,7 +112,13 @@ func OpenCTF(ctx context.Context, path string, format FileFormat, flag int) (CTF
 			return nil, fmt.Errorf("unable to hash path to determine temporary ctf: %w", err)
 		}
 
-		tmp, err := os.MkdirTemp("", fmt.Sprintf("ctf-%x-*", hash.Sum(nil)))
+		// Use filesystem configuration for temporary directory
+		tempDir := os.TempDir()
```

Comment:
super tiny nit: imo we should not initialize this variable by default as it may do an env lookup.

Reply:
Fair 👍